src/qfvm_clifford/clifford_simulator.h

```diff
@@ -155,6 +156,16 @@ template <size_t word_size> struct circuit_simulator {
 
   // get the measurement record
   auto current_measurement_record() const { return sim_record; }
+
+  auto measure_all() {
+    measurement_record mr;
+    for (size_t i = 0; i < sim_tableau.num_qubits; i++) {
+      auto record = sim_tableau.m_gate(rng, i);
+      if (record.has_value())
+        mr.record(record.value());
+    }
+    return mr;
+  }
```

Comment:
Just a very minor issue, the indents should be unified with other codes

Reply:
add pre-commit for format problem?